CLAUDE.md

```
@@ -0,0 +1,310 @@
+| Monorepo | Turborepo 2.7 |
+| Testing | Vitest (marketing), Jest (docs, ui), cargo test (Rust), pytest (Python) |
+| JS Linting | ESLint 8, Prettier 3.7 |
+| Rust Linting | Clippy, cargo fmt |
+| Python Linting | Ruff, Black, isort, mypy, bandit |
+| CI/CD | GitHub Actions, Azure Static Web Apps |
+| Styling | Tailwind CSS 4.1, CSS Modules |
+| Infrastructure | Azure Bicep, Terraform (ML training) |
+
+## Repository Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)